packages/AI/Providers/Vectors-Pinecone/src/models/PineconeDatabase.ts

```diff
@@ -186,17 +186,19 @@ export class PineconeDatabase extends VectorDBBase {
             response.Duplicates = [];
             for(const match of queryResponse.data.matches){
                 const record: {id: string, score: number, metadata: any} = match;
-                if(!record.metadata || !record.metadata.PrimaryKeys){
+                if(!record || !record.id){
```

Comment:
@js-bc - is this something from Pinecone, hence the harcoded record.id?

Reply:
Yes